doc/MANUAL.txt

```diff
@@ -451,6 +451,14 @@ might be incorrect.
     matching the compiler name in the normal *PATH* that isn't a symbolic link
     to ccache itself.
 
+*pch_external_checksum* (*CCACHE_PCH_EXTSUM*)::
```

Comment:
Please change the parenthesis content to the same format as the other boolean settings.